[DOCS] Adds content to the KSPM section (#2584)
elastic/security-docs: docs/dashboards/cloud-posture.asciidoc

* Configuration risk metrics for all monitored clusters
* Configuration risk metrics for individual clusters
* Configuration risks grouped by CIS section

[discrete]
[[cloud-posture-dashboard-UI]]
=== Cloud Posture dashboard UI

The first row of cards (Cloud Posture Score, Failed Findings, and Open Cases) summarizes your overall cloud security posture (CSP) by aggregating data from all monitored Kubernetes clusters. Each subsequent row summarizes the CSP of an individual Kubernetes cluster.

The Cloud Posture Score card shows the collective performance of all your monitored clusters. Hover over the card to display when the data was collected.

The Failed Findings card shows failed findings grouped by Center for Internet Security (CIS) benchmark categories. Click any section name to view its failed findings on the <<findings-page, Findings page>>.

Each row of cards after the first summarizes an individual cluster's CSP. Each row includes the cluster's name, the last 6 digits of its ID, its compliance score, and failed findings grouped by CIS section. When you begin to monitor a new cluster, a new row of cards appears.

[discrete]
[[cloud-posture-dashboard-faq]]
=== FAQ

> *How long after a new cluster is enrolled will it appear on the dashboard?*

Reviewer: ```suggestion
*When does a newly-enrolled cluster appear on the dashboard?*
```

It can take up to 10 minutes for deployment, resource fetching, evaluation, and data processing before an enrolled cluster appears on the posture dashboard.

> *How long after a cluster gets unenrolled will it disappear from the dashboard?*

Reviewer: ```suggestion
*When are unenrolled clusters removed from the dashboard?*
```

[[cloud-posture-dashboard]]

> // Note: This page is intentionally duplicated by docs/cloud-native-security/cloud-nat-sec-posture.asciidoc. When you update this page, update that page to match. And careful witht the anchor links because they should not match.

Reviewer 2: OK, see the previous incarnation of this content for any edits, and apply those here too.

Also fixed a li'l typo in the comment:
```suggestion
// Note: This page is intentionally duplicated by docs/cloud-native-security/cloud-nat-sec-posture.asciidoc. When you update this page, update that page to match. And careful with the anchor links because they should not match.
```

= Cloud Posture dashboard

The Cloud Posture dashboard summarizes how compliant your Kubernetes configuration is with Center for Internet Security (CIS) security benchmarks. To learn how to collect this data, refer to <<get-started-with-kspm, Get started with KSPM>>.

[role="screenshot"]
image::images/cloud-sec-dashboard.png[The cloud Security dashboard]

The Cloud Posture dashboard shows:

* Configuration risk metrics for all monitored clusters
* Configuration risk metrics for individual clusters
* Configuration risks grouped by CIS section

It enables you to assess your infrastructure's overall performance against CIS <<benchmark-rules,security benchmarks>>, and to investigate failed findings.

[discrete]
[[cloud-posture-dashboard-UI]]
=== Cloud Posture dashboard UI

The first row of cards (Cloud Posture Score, Failed Findings, and Open Cases) summarizes your overall cloud security posture (CSP) by aggregating data from all monitored Kubernetes clusters. Each subsequent row summarizes the CSP of an individual Kubernetes cluster.

> The Cloud Posture Score card shows the collective performance of all your monitored clusters. Hover over the card to display when the data was collected.

Reviewer 3: ```suggestion
The Cloud Posture Score card shows the collective performance of all your monitored clusters over time.
```
